events/wc_beyond_dp.txt

```diff
@@ -100,10 +100,6 @@ narrative_event = {
 	}
 	
 	is_triggered_only = yes
-	
-	immediate = {
-		character_event = { id = WCSDW.3 }		# Creates the Warsong Clan
-	}
```

Comment:
Emm, why it's removed?

Reply:
Because there's no real point to spawn a second one anymore. 

As per my main post:
```
The second spawned Grommash, which has caused confusion before, is no longer getting spawned. There's just one Grommash now, but it's easy enough to implement him no longer having the trait earlier so this behaviour can be emulated with the actual historical char.
```

If you'd prefer for him to lose the `on_another_world` trait earlier around that time, it's as simple as doing the `remove_trait` within that immediate block and then giving him an army so he can raid as-is. But it seems unnecessary to go the "double spawn" route again.

Reply:
With how the `On Another World` is coded he can't really die though, the `health = 100` kind of makes that near impossible, no?

Reply:
Not really, natural death chance grows exponentially. So, even 100 health can't save you if you're 100 years old.

Reply:
Sure, but this can't happen to Grommash as he returns at around 34 years of age or so.

The only ones who stay away for 'thousands' years are Turalyon and Alleria. But they're not supported yet by this.. and probably won't ever be, I'm gonna guess (since that's Legion).

Reply:
Wait, who said you can't die because of natural reason at 34? Death is always random.

Reply:
Sure, but isn't that where the 100 health comes into? I feel like 100 health with a < 40 year old is near impossible to touch.

Together with all those `is_untouchable_trigger = yes` checks I figured they'd be mostly safe. But I'm gonna guess CK2 loves to play hardcore and there's still methods?

Reply:
There's always chance of Grommash's death., you know.